script/int-log-compiler.sh

```diff
@@ -54,6 +54,23 @@ while test $# -gt 0; do
     shift
 done
 
+sanity_test ()
+# Verify the running shell and OS environment is sufficient to run these tests.
```

Comment:
This is a first for me. Never seen anyone put comments between the closing parenthesis of a function prototype and the opening brace of the function body.

I guess we don't explicitly disallow this, and we don't have a section of the coding standard for shell scripts, but there are ample examples in this file of functions and comments describing them. Please use the same convention. Also see the final sentence of our coding: https://github.com/tpm2-software/tpm2-tss/blob/master/doc/coding_standard_c.md#coding-standard specifically:
```
Above all else we ask that when modifying the code that you
follow the most important rule: check the surrounding code and
imitate its style.
```

Reply:
Accepted. It is confusing to have comments about a function before the function name, but if that is the project style I can do that.